src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,70 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+/* eslint-disable jsx-a11y/control-has-associated-label */
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  filterType: string;
+  setFilterType: (str: string) => void;
+  query: string;
+  setQuery: (str: string) => void;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = ({
+  filterType,
+  setFilterType,
+  query,
+  setQuery,
+}) => {
+  const handleChangefilterType
+  = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
code style 😭
```suggestion
  const handleChangefilterType = (
    event: React.ChangeEvent<HTMLSelectElement>,
  ) => {
```

Reply:
hmmm... it's strange, but in my vs-code, the code style is written according to your comment, but here it goes like this